packages/inputs-tiny/src/radio/index.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import styles from './index.module.scss';
+
+const Radio = ({
+  onChange,
+  className,
+  disabled,
+  value,
+  label,
+  selectedValue,
+  ...rest
+}) => {
+  const handleChange = (e) => {
+    onChange(value.toString(), e);
+  };
+
+  return (
+    <label className={clsx(styles.label, { [styles.disabled]: disabled })}>
+      <input
+        {...rest}
```

Comment:
Do we continue with passing `rest` on to the `input`? Makes it hard to refactor later. Are there downsides on not doing it? (data attributes?)